applications/solvcon/prepare-solvcon-dev.sh

```diff
@@ -6,7 +6,14 @@
 #   source <this script> <your-project-folder>
 #
 set -x
-SOLVCON_PROJECT=${1:-${HOME}/solvcon}
+
+if [ -z "${DEVENVFLAVOR}" ]
+then
+  SOLVCON_PROJECT=${1:-${HOME}/solvcon}
+else
+  SOLVCON_PROJECT=${1:-${DEVENVAPPLICATION}}
```

Comment:
I think `DEVENVAPPLICATION` is unnecessarily long.  How about `DEVENVAPP`?

Reply:
Excellent name!